src/Dapr.Workflow/WorkflowServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2022 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+namespace Dapr.Workflow
+{
+    using System;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.DurableTask.Client;
+    using Microsoft.DurableTask.Worker;
+
+    /// <summary>
+    /// Contains extension methods for using Dapr Workflow with dependency injection.
+    /// </summary>
+    public static class WorkflowServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds Dapr Workflow support to the service collection.
+        /// </summary>
+        /// <param name="serviceCollection">The <see cref="IServiceCollection"/>.</param>
+        /// <param name="configure">A delegate used to configure actor options and register workflow functions.</param>
+        public static IServiceCollection AddWorkflow(
```

Comment:
`AddDaprWorkflow()`?

Reply:
I could go either way, but the Actors SDK names the equivalent extension method `AddActors()` (no "Dapr") so `AddDaprWorkflow` would be a bit inconsistent with the actor SDK.

Reply:
I'm fine either way. I'm not super familiar with the existing options though, so my concern would be if this would clobber anything. Do we expect people to have our libraries and the other workflow libraries together?